stores/userStore.ts

```
@@ -0,0 +1,88 @@
+	User,
+	NewUserInput,
+	LoginUserInput,
+	LoginUser,
+} from "@/types/userTypes";
+
+interface UserState {
+	user: User | LoginUser | null;
+	status: "idle" | "loading" | "succeeded" | "failed";
+	error: unknown | null;
+
+	login: (userData: LoginUserInput) => Promise<any>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)